.storybook/config.js

```diff
@@ -1,18 +1,26 @@
 import React, { Fragment } from 'react';
 import { configure, addDecorator, addParameters } from '@storybook/react';
 import { create } from '@storybook/theming';
-import GlobalStyle from '../src/app/lib/globalStyles';
+import * as fontFaces from '@bbc/psammead-styles/fonts';
+import { createGlobalStyle } from 'styled-components';
 
 const req = require.context('../src/app', true, /\.stories\.jsx$/);
 
 function loadStories() {
   req.keys().forEach(filename => req(filename));
 }
 
+const styles = Object.keys(fontFaces).reduce(
+  (fontStyles, face) => fontFaces[face] + fontStyles,
+  '',
+);
+
+const GlobalStyles = createGlobalStyle`${styles}`;
```

Comment:
This removes the `styledNormalize` and box sizing stuff from https://github.com/bbc/simorgh/blob/latest/src/app/lib/globalStyles/index.jsx which will technically mean css is different between storybook & pages, is that intended?

Could we maybe make https://github.com/bbc/simorgh/blob/latest/src/app/lib/globalStyles/index.jsx reusable? Maybe with
```
const { fonts } = props || useContext(ServiceContext);
```